includes/admin/class-wc-stripe-admin-notices.php

```diff
@@ -558,6 +618,24 @@ public function stripe_updated() {
 			update_option( 'wc_stripe_show_sca_notice', 'no' );
 		}
 	}
+
+	/**
+	 * Checks if the current page is a subscription edit page in wp-admin.
+	 *
+	 * This should be removed once WooCommerce provides a way to check for subscription edit pages.
+	 *
+	 * @return bool
+	 */
+	private static function is_subscription_edit_page() {
+		$query_params = wp_unslash( $_GET ); // phpcs:ignore WordPress.Security.NonceVerification.Missing
```

Comment:
Might it be worth checking `$_REQUEST` to handle `POST` requests to the page as well?
```suggestion
		$query_params = wp_unslash( $_REQUEST ); // phpcs:ignore WordPress.Security.NonceVerification.Missing
```